devel/libprotozero/Portfile

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem                                    1.0
+PortGroup           cmake                     1.1
+PortGroup           github                    1.0
+
+name                libprotozero
+
+github.setup        mapbox protozero 1.7.0 v
+
+categories          devel
+platforms           darwin
+maintainers         {@frankdean fdsd.co.uk:frank.dean} openmaintainer
+license             BSD Boost-1 Apache-2
```

Comment:
the way you specified the licenses here implies that all three license apply at the same time. I couldn't really find that information easily upstream, but I am sure you got them from somewhere. If on the other hand you have a choice and can pick one of the three you should surround them with `{}`.

Reply:
It's primarily licensed under BSD, but some embedded code is from other sources with the Boost and Apache licences.  I've commented the Portfile with references.

Reply:
I didn't mean that you needed to add comments about this to the Portfile. I was just pointing out the difference between listing three of them as ` BSD Boost-1 Apache-2` or `{ BSD Boost-1 Apache-2}` so that you could pick the correct one. So from your description I agree that the way you have it is correct and you don't need the comments.

Reply:
I've removed the comments.